docs/howto/graph-api/query.archive.md

```diff
@@ -0,0 +1,706 @@
+# Querying Archive
+
+In the previous page, we explained how to fetch data from different data resources (mainly `MongoDB` documents).
+They are mostly flat (with only a few levels of nesting) and relatively easy to query and fetch.
+
+The very idea can be extended to fetching archives that are stored on the file system.
+
+## Accessing Archives
+
+An archive is the processed data of an entry, which is stored on the file system as a binary file.
+Each archive thus corresponds to an entry, and the corresponding entry ID can be used as the unique identifier to access the archive.
+In the graph system, the archive is linked to the corresponding entry via the special token `archive`.
+Thus, to access the archive of an entry with ID `example_entry_id`, one can use the following query.
+
+```json hl_lines="4-6"
+{
+   "entries":{
+      "example_entry_id":{
+         "archive":{
+            "m_request":{ "directive":"plain" }
```

Comment:
The `"directive": "plain"`, is a bit mysterious for the reader at this point. Is it the default behaviour? If so, maybe we don't need to include it in these examples? If you think it is better to keep it here, add a small note + link to the `"Resolving References"` section that explains the meaning of this in more detail.